app/controllers/application_controller.rb

```diff
@@ -1,5 +1,14 @@
 class ApplicationController < ActionController::Base
-  def current_user
-    User.first
+  protect_from_forgery with: :exception
+
+  before_action :update_allowed_parameters, if: :devise_controller?
+
+  protected
+
+  def update_allowed_parameters
+    devise_parameter_sanitizer.permit(:sign_up) { |u| u.permit(:name, :posts_counter, :email, :password) }
```

Comment:
- [ ] I suggest you include `password_confirmation` as one of the permitted parameter because I received this error below;

<img width="419" alt="Code_hcNnvZQxqR" src="https://user-images.githubusercontent.com/81468675/156156817-c36012e7-9de8-4def-b516-dceafa2182b7.png">

Reply:
Thanks noted, will fix ASAP